fml/file.cc

```diff
@@ -93,3 +93,20 @@ fml::UniqueFD OpenDirectoryReadOnly(const fml::UniqueFD& base_directory,
 }
 
 }  // namespace fml
+
+void fml::RemoveFilesInDirectory(const fml::UniqueFD& directory) {
```

Comment:
Move this inside the `namespace fml` block

Reply:
Done.